soc/arm/st_stm32/stm32g4/Kconfig.defconfig.stm32g474re

```diff
@@ -6,11 +6,9 @@
 if SOC_STM32G474XX
 
 config SOC
-	string
 	default "stm32g474xx"
 
 config NUM_IRQS
-	int
 	default 102
```

Comment:
Let's split this is a dedicated commit (same PR could be used).

Reply:
done into 13c405dda1e2f443a592a5464120fdf04cddbf84 and  3611b5da65c826dd2d62ac3d2517f316a0670040